pywb/rewrite/wburl.py

```diff
@@ -248,6 +255,26 @@ def _init_query(self, url):
             self.type = self.QUERY
         return True
 
+    # Match continuity regex
+    # ======================
+
+    def _init_continuity(self, url):
+        continuity = self.CONTINUITY_REGEX.match(url)
+        if not continuity:
+            return None
+
+        res = continuity.groups('')
+
+        self.timestamp = res[0]
+        self.mod = res[1]
+        self.url = res[2]
+        if self.url.endswith('*'):
+            self.type = self.CONTINUITY
+            self.url = self.url[:-1]
```

Comment:
Why do we need to do `self.url[:-1]` to remove the last character of the URL?

Reply:
Basically, it's removing the '*' from the end of the URL. 
if the URL looks like this `/+/https://www.agender.org.uk/*` it will remove the '*' to get the original URL. 
Here I've followed the old PYWB working maybe someone mistakenly put the * at the end of URL or there are some URL in TNA dir ends with *